testsuite/MDAnalysisTests/converters/test_openmm_parser.py

```diff
@@ -130,6 +150,44 @@ class TestOpenMMTopologyParser(OpenMMTopologyBase):
     expected_n_bonds = 1922
 
 
+class TestOpenMMTopologyParserWithNoElements(OpenMMTopologyBase):
+    ref_filename = app.PDBFile(PDB).topology
+    expected_n_atoms = 47681
+    expected_n_residues = 11302
+    expected_n_segments = 1
+    expected_n_bonds = 25533
+
+    def test_with_partial_elements(self, top):
+        if 'elements' in self.expected_attrs:
+            omm_topology = self.ref_filename
+            wmsg = ("Unknown element None found for some atoms. "
+                    "These have been given an empty element record "
+                    "with their atomtype set to 'X' "
+                    "and their mass set to 0.0. "
+                    "If needed they can be guessed using "
+                    "MDAnalysis.topology.guessers.")
+            with pytest.warns(UserWarning, match=wmsg):
+                mda_top = self.parser(PDB) \
+                          ._mda_topology_from_omm_topology(omm_topology)
```

Comment:
Could you please make this a separate class to the test for no elements, and then actually check a couple output values? e.g. if `mda_top.elements[9] == "X"`.